### PR TITLE
People page

### PR DESCRIPTION
**Pull request by bluefirerose**
Hi Rei, 

Thank you for reviewing my code. I have made all the updates you have noted on your email. 
-Updated the bootstrap to the latest version. Updated all the affected bootstrap tags. 
-Added card and container to wrap the artist's info. 
-Fixed the horizontal bar for movie list like the figma. 

Feel free to keep me posted for any changes you recommend.

### CODE REVIEW

File: peoplepage.html [hunk 1]
```diff
@@ -59,15 +85,15 @@
                         <li class="nav-item dropdown px-lg-3">
                             <a class="nav-link active dropdown-toggle" href="" role="button" data-bs-toggle="dropdown">Categories</a>
                               <ul class="dropdown-menu" id="categoryItems">
```
Comment (reiurbano) [inline peoplepage.html:87]: missing some javascript to add the category list items

File: peoplepage.html [hunk 2]
```diff
@@ -77,138 +103,141 @@
...
+        function displayArtists(artists) {
+            artistsContainer.innerHTML = '';
+
+            if (artists.length === 0) {
+                const artistNotFoundMessage = document.createElement('p');
+                artistNotFoundMessage.textContent = 'Artist not found.';
+                artistsContainer.appendChild(artistNotFoundMessage);
+            } else {
+                let row = document.createElement('div');
+                row.classList.add('row');
```
Comment (reiurbano) [inline peoplepage.html:156]: suggest to add  a justify-content-center class here to center the artists on the page.

File: peoplepage.html [hunk 3]
```diff
@@ -77,138 +103,141 @@
...
+                        const artistInfoUrl = `https://api.themoviedb.org/3/person/${artistId}?api_key=${apiKey}`;
+                        window.location.href = `artist.html?id=${artistId}&url=${artistInfoUrl}`;
+                    });
+
+                    row.appendChild(artistCard);
+
+                    if ((index + 1) % 5 === 0) {
+                        artistsContainer.appendChild(row);
+                        row = document.createElement('div');
+                        row.classList.add('row');
```
Comment (reiurbano) [inline peoplepage.html:184]: same as above, adding the class to center the artist elements

File: peoplepage.html [hunk 4]
```diff
@@ -5,22 +5,38 @@
...
+            width: 100%;
+            height: 80%;
             object-fit: cover;
             margin: 0px;
             margin-bottom: 10px;
             border-radius: 9px;
+            margin-top: 10px;
         }
 
         nav{
```
Comment (reiurbano) [inline peoplepage.html:42]: change  from nav to header to remove the uncovered  part on the left of the navbar

File: peoplepage.html [hunk 5]
```diff
@@ -5,22 +5,38 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.3.0/css/bootstrap.min.css">
     <style>
+        @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+        * {
+            font-family: 'Poppins', sans-serif;
+            margin: 0;
+            padding: 0;
+        }
+        .artist{
+            margin-left: 65px;
```
Comment (reiurbano) [inline peoplepage.html:17]: change from margin-left to margin-right: auto; to better prevent the search bar from clipping

File: peoplepage.html [hunk 6]
```diff
@@ -30,9 +46,19 @@
         .searchbox {
             margin-left: 750px;
```
Comment (reiurbano) [inline peoplepage.html:47]: same here change to margin-left: auto to prevent clipping

File: artist.html
Comment (reiurbano) [inline artist.html:1]: suggest  changing the container colors alternating between either black or white, centering the elements, theres also some overlapping between the profile and the personal information box on certain screen sizes, and some of personal info overflows out of the container.

File: peoplepage.html (hunk 6 above)
Comment (bluefirerose) [reply peoplepage.html:47]: done

File: peoplepage.html (hunk 5 above)
Comment (bluefirerose) [reply peoplepage.html:17]: done

File: peoplepage.html (hunk 4 above)
Comment (bluefirerose) [reply peoplepage.html:42]: done

File: peoplepage.html (hunk 3 above)
Comment (bluefirerose) [reply peoplepage.html:184]: done

File: peoplepage.html (hunk 2 above)
Comment (bluefirerose) [reply peoplepage.html:156]: done

File: artist.html
Comment (bluefirerose) [reply artist.html:1]: Done. Kindly check if it is okay.

File: peoplepage.html (hunk 1 above)
Comment (bluefirerose) [reply peoplepage.html:87]: in progress...
stuck to this script.

File: artist.html
Comment (reiurbano) [reply artist.html:1]: if think having the personal details and movie section be the white part while the biography in the middle is the dark container be better.

File: peoplepage.html (hunk 1 above)
Comment (reiurbano) [reply peoplepage.html:87]: `        // Get Genres
        const fetchGenreData = () => {
            fetch('https://api.themoviedb.org/3/genre/movie/list', options)
                .then(response => response.json())
                .then(response => {
                    response = response.genres;
                    response.forEach((result) => {
                        //dNavbar Category Display
                        document.querySelector("#categoryItems").innerHTML += `
                            <li>
                                <a class="dropdown-item" href="./search.html?list=${result.id}">${result.name}</a>
                            </li>
                        `;
                    });
                })
        }

        fetchGenreData();
`

Comment (reiurbano) [reply peoplepage.html:87]: `
        const fetchGenreData = () => {
            fetch('https://api.themoviedb.org/3/genre/movie/list', options)
                .then(response => response.json())
                .then(response => {
                    response = response.genres;
                    response.forEach((result) => {
                        //dNavbar Category Display
                        document.querySelector("#categoryItems").innerHTML += `
                            <li>
                                <a class="dropdown-item" href="./search.html?list=${result.id}">${result.name}</a>
                            </li>
                        `;
                    });
                })
        }
        fetchGenreData();
`

Comment (bluefirerose) [reply peoplepage.html:87]: Done. Kindly check if it works.

**Pull request merged**